cms/djangoapps/contentstore/tasks.py

```diff
@@ -464,7 +464,8 @@ def sync_discussion_settings(course_key, user):
             course.discussions_settings['provider_type'] = Provider.OPEN_EDX
             modulestore().update_item(course, user.id)
 
```

Comment:
should we remove this blank line?

Reply:
I intentionally kept it to separate logical blocks i.e. modulestore update and DiscussionConfiguration.